indico/modules/events/timetable/templates/_timetable.html

```diff
@@ -43,9 +43,13 @@
                 new BrowserHistoryBroker(),
             ].concat(extraArgs);
 
-            var timetable = new (Function.prototype.bind.apply(timetableClass, timetableArgs));
-            $('#timetable').html(timetable.draw());
-            timetable.postDraw();
+            if (Object.getOwnPropertyNames(timetableArgs[1]).length > 0) {
+                var timetable = new (Function.prototype.bind.apply(timetableClass, timetableArgs));
+                $('#timetable').html(timetable.draw());
+                timetable.postDraw();
+            } else {
+                $('#timetable').html($T("The timetable has not been filled yet."));
```

Comment:
I'm not sure if I've understood your comment correctly. Do you mean the buttons to add a contribution/break/session block in `/event/<event_id>/manage/timetable/`. I didn't find anything wrong with them.

This is also used in management but it's not an issue since the timetable won't be empty (it will contain empty days).